src/engraving/layout/v0/tlayout.cpp

```diff
@@ -4222,10 +4222,10 @@ void TLayout::layout(StemSlash* item, LayoutContext& ctx)
 
     double up = c->up() ? -1 : 1;
     double stemTipY = c->up() ? stem->bbox().translated(stem->pos()).top() : stem->bbox().translated(stem->pos()).bottom();
-    double leftHang = ctx.conf().noteHeadWidth() * ctx.conf().styleD(Sid::graceNoteMag) / 2;
+    double leftHang = ctx.conf().noteHeadWidth() * c->mag() / 2;
```

Comment:
You should not call this c->mag() twice. Call it only once, save the result to a temp variable, and reuse it twice

Reply:
done!